src/main/kotlin/no/nav/dagpenger/streams/Service.kt

```diff
@@ -17,7 +17,7 @@ import java.util.Properties
 
 private val LOGGER = KotlinLogging.logger {}
 
-abstract class Service {
+abstract class Service(private val username: String? = null, private val password: String? = null) {
```

Comment:
not sure about this.. Should we refactor out kafka config as a separate class? 
Reason for injecting and not using `getenv("KAFKA_USERNAME")` is that every service need to put 

```
  - alias: srvdagpenger-joark-mottak
    resourceType: credential
    propertyMap:
      username: KAFKA_USERNAME
      password: KAFKA_PASSWORD
```

instead of just 

```
  - alias: srvdagpenger-joark-mottak
    resourceType: credential
```
and use `getEnv("SRVJOARKMOTTAK_USERNAME")` 

We need to reuse username and password for eg. sts

Reply:
The idea was that one could override the `getConfig()` method and extend the set of configuration in service that needs it.

```suggestion
abstract class Service() {
```